source/reference/mongosync-behavior.txt

```diff
@@ -44,107 +47,9 @@ line options.
    - For a full list of limitations, see :ref:`Cluster-to-Cluster Sync
      Limitations<c2c-limitations>`. 
 
-.. _c2c-cli-options:
-
-Options
--------
-
-.. |opt-term| replace:: option
-
-Global Options
-~~~~~~~~~~~~~~
-
-.. option:: --cluster0 <URI>
-
-   .. include:: /includes/opts/cluster0.rst
-
-   To set the ``--cluster0`` option from a configuration file,
-   see the :setting:`cluster0` setting.
-
-.. option:: --cluster1 <URI>
-
-   .. include:: /includes/opts/cluster1.rst
-
-   To set the ``--cluster1`` option from a configuration file,
-   see the :setting:`cluster1` setting.
-
-.. option:: --config <filename>
-
-   Sets the path to the configuration file.
-
-   For more information, see :ref:`c2c-mongosync-config`.
-
-.. option:: --disableTelemetry
-
-   .. versionadded:: 1.4.0
-
-   .. include:: /includes/opts/disableTelemetry
-
-   To set the ``--disableTelemetry`` option from a configuration file,
-   see the :setting:`disableTelemetry` setting.
-
-   For more information, see :ref:`User Data Collection
-   <c2c-telemetry>`.
-
-.. option:: --help, -h
-
-   Prints usage information to stdout.
-
-.. option:: --id <ID>
-
-   .. include:: /includes/opts/id.rst
-
-   To set the ``--id`` option from a configuration file,
-   see the :setting:`id` setting.
-
-.. option:: --loadLevel <level>
-
-   *Type*: integer
-
-   .. include:: /includes/opts/loadLevel.rst
-
-   To set the ``--loadLevel`` option from a configuration file, see the
-   :setting:`loadLevel` setting.
-
-   .. include:: /includes/opts/loadlevel-warning.rst
-   
-.. option:: --logPath <DIR>
-
-   .. include:: /includes/opts/logPath.rst
-
-   To set the ``--logPath`` option from a configuration file,
-   see the :setting:`logPath` setting.
-
-   .. note:: 
-
-      .. include:: /includes/fact-log-rotation-usr1-signal
-
-.. option:: --port
-
-   .. include:: /includes/opts/port.rst
-
-   To set the ``--port`` option from a configuration file,
-   see the :setting:`port` setting.
-
-.. option:: --verbosity <level>
-
-   .. |verbosity-opt-type| replace:: option
-   .. |verbosity-opt| replace:: ``--verbosity``
-
-   .. include:: /includes/opts/verbosity.rst
-
-   To set the ``--verbosity`` option from a configuration file,
-   see the :setting:`verbosity` setting.
-   
-.. option:: --version, -v
-
-   Prints ``mongosync`` version information to stdout.
-
 Behavior
```

Comment:
Since the title of this page is already "Behavior", I think we can remove this heading a bump ever other heading up (ie, making all the H3s into H2s and so on)

Reply:
Good idea, doing now!